sympy/printing/theanocode.py

```diff
@@ -132,6 +131,14 @@ def _print_MatMul(self, expr, **kwargs):
             result = tt.dot(result, child)
         return result
 
+    def _print_MatPow(self, expr, **kwargs):
+        children = [self._print(arg, **kwargs) for arg in expr.args]
+        result = 1
+        if isinstance(children[1], int):
```

Comment:
What is the power is negative?

Reply:
Good point, should check that it's not.
Ideally, it would be nice to be able print any power but I couldn't work out how to do it in theano. `tt.pow` that is used for power of symbolic expressions doesn't work on tensor objects and the documentation on tensors didn't really offer an alternative.